src/status_im/contexts/onboarding/enable_notifications/style.cljs

```diff
@@ -24,3 +24,20 @@
   [insets]
   {:margin        default-margin
    :margin-bottom (+ 14 (:bottom insets))})
+
+(def background-image
+  {:position :absolute
+   :top      0
+   :bottom   0
+   :left     0
+   :right    0})
```

Comment:
anywhere you are using this style, you can instead replace it by referencing `rn/stylesheet-absolute-fill` which is a JS-obj that won't be parsed by reagent.

Reply:
if you are merging this style, you can just use a vector so RN merges it, e.g.:
```clojure
[rn/view {:style [rn/stylesheet-absolute-fill {:padding 20}] 
 ...]
```

Reply:
In a future, it'd be great to reduce our redundant styles, I wonder if that will improve our loading time and rendering time